hedera-node/test-clients/README.md

```diff
@@ -1,249 +1,282 @@
+   without access to their block streams, internal state, or file system.
 2. `SubProcessNetwork` - a managed network of four child processes, each running a Hedera node. This
-    implementation supports starting and stopping nodes, and provides access to each node's block stream,
-    logs, and upgrade artifacts. However, it does not support direct access to working state.
+   implementation supports starting and stopping nodes, and provides access to each node's block
+   stream, logs, and upgrade artifacts. However, it does not support direct access to working state.
 3. `ConcurrentEmbeddedNetwork` - a simulated network that instantiates a single `Hedera` instance
    directly in the test process. The internal state is a `FakeHederaState` implementation whose data
-   sources are collections from the `java.util.concurrent` package. This allows direct access to, and
-   modification of, the network's state.
+   sources are collections from the `java.util.concurrent` package. This allows direct access to,
+   and modification of, the network's state.
 4. `RepeatableEmbeddedNetwork` - an embedded variant automatically selected when running a
-   `HapiSpec` in repeatable mode. This implementation requires single-threaded test execution and uses
-   virtual time to ensure the same consensus times every test run.
+   `HapiSpec` in repeatable mode. This implementation requires single-threaded test execution and
+   uses virtual time to ensure the same consensus times every test run.
 
-Examining the differences between these implementations as we move from black box testing to increasingly
-white box testing provides valuable insights into the key architectural elements of a Hedera network.
-Let’s take a closer look.
+Examining the differences between these implementations as we move from black box testing to
+increasingly white box testing provides valuable insights into the key architectural elements of a
+Hedera network. Let’s take a closer look.
 
 ### `RemoteNetwork`
 
-A `HapiSpec` executing against a `RemoteNetwork` behaves like any other client of that network. It sends
-transactions and queries to the network’s gRPC services and receives responses as protobuf messages.
-While the `RemoteNetwork` provides the most realistic testing environment for a spec, it is also the
-most limited in terms of what the test can observe and manipulate. Network latency also makes it the
-slowest way to execute a spec.
+A `HapiSpec` executing against a `RemoteNetwork` behaves like any other client of that network. It
+sends transactions and queries to the network’s gRPC services and receives responses as protobuf
+messages. While the `RemoteNetwork` provides the most realistic testing environment for a spec, it
+is also the most limited in terms of what the test can observe and manipulate. Network latency also
+makes it the slowest way to execute a spec.
 
 <img src="./docs/assets/remote-network.png" width="900">
```

Comment:
Do we need a set width here?

If not I'd convert the image to markdown syntax.